website/docs/faqs/Troubleshooting/job-memory-limits.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "I'm receiving a 'This run exceeded your account's run memory limits' error in my failed job"
+description: "Use incremental models or optimize queries for job failures due to exceeded memory limits."
+sidebar_label: 'Job failures due to exceeded memory limits'
+---
+
+If you're receiving a `This run exceeded your account's run memory limits` error in your failed job, it means that the job exceeded the memory limits set for your account. Memory limits are on a per run basis and are typically influenced by the amount of result data that dbt has to ingest and process, which typically is quite small but can be bloated unexpectedly by project design choices.
```

Comment:
add link to this faq directly in the error message: https://github.com/dbt-labs/dbt-cloud/blob/378f5a06d18998c367d1c4a3473cd607f98f2b08/sinter/common/error_reasons.py#L3